p2p/client_test.go

```diff
@@ -67,7 +67,7 @@ func TestClientStartup(t *testing.T) {
 }
 
 func TestBootstrapping(t *testing.T) {
-	log.SetLogLevel("dht", "INFO")
+	_ = log.SetLogLevel("dht", "INFO")
 	//log.SetDebugLogging()
```

Comment:
I am not sure if we need to keep that logging/debugging code in master actually. I usually remove that code before merging and put it back for local usage only without committing